specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBRestorableSqlDatabaseList.json

```diff
@@ -0,0 +1,83 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "2296c272-5d55-40d9-bc05-4d56dc2d7588",
+    "location": "WestUS",
+    "instanceId": "d9b26648-2f53-4541-b3d8-3044f4f9810d"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/2296c272-5d55-40d9-bc05-4d56dc2d7588/providers/Microsoft.DocumentDb/locations/westus/restorableDatabaseAccounts/36f09704-6be3-4f33-aa05-17b73e504c75/restorableSqlDatabases/59c21367-b98b-4a8e-abb7-b6f46600decc",
+            "type": "Microsoft.DocumentDB/locations/restorableDatabaseAccounts/restorableSqlDatabases",
+            "name": "59c21367-b98b-4a8e-abb7-b6f46600decc",
+            "properties": {
+              "resource": {
+                "_rid": "DLB14gAAAA==",
+                "eventTimestamp": "2020-09-02T19:45:03Z",
+                "ownerId": "Database1",
+                "ownerResourceId": "3fu-hg==",
+                "operationType": "Create",
+                "database": {
+                  "id": "Database1",
+                  "_rid": "3fu-hg==",
+                  "_self": "dbs/3fu-hg==/",
+                  "_etag": "\"0000c20a-0000-0700-0000-5f4ff63f0000\"",
+                  "_colls": "colls/",
+                  "_users": "users/"
+                }
+              }
+            }
+          },
+          {
+            "id": "/subscriptions/2296c272-5d55-40d9-bc05-4d56dc2d7588/providers/Microsoft.DocumentDb/locations/westus/restorableDatabaseAccounts/d9b26648-2f53-4541-b3d8-3044f4f9810d/restorableSqlDatabases/8456cb17-cdb0-4c6a-8db8-d0ff3f886257",
+            "type": "Microsoft.DocumentDB/locations/restorableDatabaseAccounts/restorableSqlDatabases",
+            "name": "8456cb17-cdb0-4c6a-8db8-d0ff3f886257",
+            "properties": {
+              "resource": {
+                "_rid": "ESXNLAAAAA==",
+                "eventTimestamp": "2020-09-02T19:53:42Z",
+                "ownerId": "Database1",
+                "ownerResourceId": "3fu-hg==",
```

Comment:
are these really required ? In every case, these values will be equal to database.id and database._rid. Why would we return two objects which represent the same concept.

Reply:
For Mongo, we don't have the "Content" properties, so we need to expose it here.  Doing the same for Sql APIs for consistency